inst/resources/bs4_book/bs4_book.css

```diff
@@ -456,7 +456,7 @@ code span.cv {color:#545454; font-style: italic}
 code span.cn {color:#d91e18}
 code span.sc {color:#008000}
 code span.dv {color:#a1024a}
-code span.kw {color:#007faa;}
+code span.kw {color:#007faa; font-weight: bold;}
```

Comment:
This will impact default look on current bs4_book() as `kw` style is used with R already. 

Is this important to have bold ?

Reply:
@cderv Bolding would bring the syntax highlighting style more in line with what Pygments does, but I don't think it's necessary; just consistent with a particular style.

I removed the bolding in a7832dac3bf5586117026514ff33c90f1100eb8a